client/account-recovery/style.scss

```diff
@@ -0,0 +1,6 @@
+.account-recovery__container {
+	margin: auto;
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/8955/commits/f711cfe9dc59c8dee757445809569bfbdee8b8d4